src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task {
+
+    public Event(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        String[] dates =  description.split("/");
```

Comment:
You could shorten this comversion by splitting based off multiple delimiters.
You can find more info here: https://stackoverflow.com/questions/5993779/use-string-split-with-multiple-delimiters